devices/update.js

```diff
@@ -40,6 +40,24 @@ module.exports.update = (event, context, callback) => {
     });
 
     dynamodb.update({
+<<<<<<< HEAD
```

Comment:
This is conflict with master branch. Please pull latest code from master to your branch